packages/pieces/community/text-helper/src/lib/actions/replace.ts

```diff
@@ -9,6 +9,7 @@ export const replace = createAction({
     text: Property.ShortText({
       displayName: 'Text',
       required: true,
+      validators: [Validators.string]
```

Comment:
It's better to add to defaultValidator directly inside the piece framework, another way also to add defaultProcessor where it convert it to a string.